pkg/gohai/cpu/cpu_windows.go

```diff
@@ -10,9 +10,10 @@ import (
 	"regexp"
 	"strconv"
 	"strings"
-	"syscall"
 	"unsafe"
 
+	syscall "golang.org/x/sys/windows"
```

Comment:
Do we want to keep the package renamed like before (`syscall`)? Might be better to use the new name (`windows`) to make it obvious that these are win-specific functions but it's up to you.